app/components/pages-menu/style.scss

```diff
@@ -6,3 +6,23 @@
     border-radius: 2px;
     text-decoration: none;
 }
+
+.pages-nav {
+    &.left {
+       text-align: left;
+       float:left
```

Comment:
It's probably because the size of the navigation element did not extend to full width. Can you try to find out why align is not working, because that should be the right way to do it.